a_worksheet.rb

```diff
@@ -0,0 +1,145 @@
+drivers_db = {
+    DR001: [
+      {
+        date: "2016-02-03",
+        cost: 10,
+        rating: 3,
+        rider_id: "RD0003",
+      },
+      {
+        date: "2016-02-03",
+        cost: 30,
+        rating: 4,
+        rider_id: "RD0015",
+      },
+      {
+        date: "2016-02-05",
+        cost: 45,
+        rating: 2,
+        rider_id: "RD0003",
+      }
+    ],
+    DR002: [
+        {
+          date: "2016-02-03",
+          cost: 25,
+          rating: 5,
+          rider_id: "RD0073",
+        },
+        {
+          date: "2016-02-04",
+          cost: 15,
+          rating: 1,
+          rider_id: "RD0013",
+        },
+        {
+          date: "2016-02-05",
+          cost: 35,
+          rating: 3,
+          rider_id: "RD0066",
+        }
+      ],
+      DR003: [
+        {
+          date: "2016-02-04",
+          cost: 5,
+          rating: 5,
+          rider_id: "RD0066",
+        },
+        {
+          date: "2016-02-05",
+          cost: 50,
+          rating: 2,
+          rider_id: "RD0003",
+        }
+      ],
+      DR004: [
+        {
+          date: "2016-02-03",
+          cost: 5,
+          rating: 5,
+          rider_id: "RD00722",
+        },
+        {
+          date: "2016-02-04",
+          cost: 10,
+          rating: 4,
+          rider_id: "RD0022",
+        },
+        {
+          date: "2016-02-05",
+          cost: 20,
+          rating: 5,
+          rider_id: "RD0073",
+        }
+      ]
+}
+
+driver1_rides = drivers_db[:DR001]
+driver2_rides = drivers_db[:DR002]
+driver3_rides = drivers_db[:DR003]
+driver4_rides = drivers_db[:DR004]
+
+# puts "#{driver1_rides.length} rides"
+# puts "#{driver2_rides.length} rides" 
+# puts "#{driver3_rides.length} rides"
+# puts "#{driver4_rides.length} rides"
+allrides = [driver1_rides, driver2_rides, driver3_rides, driver4_rides]
```

Comment:
Very small nitpick but: to follow Ruby naming conventions more closely, `allrides` should be `all_rides`